test/compliance.spec.js

```diff
@@ -22,10 +26,17 @@ describe('compliance tests', () => {
         compat: true
       })
 
+      // Trigger discovery
```

Comment:
The interface update requires discovery to be triggered to fulfil some tests, like we do in other discovery modules like: https://github.com/libp2p/js-libp2p-webrtc-star/blob/master/test/compliance.spec.js#L54

This is not the optimal solution though, and I would like to change the testing interface in the future to return multiple discovery modules instances and start them all when needed.